src/Kentico.Xperience.Lucene/Models/IndexedItemModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Kentico.Xperience.Lucene.Models;
+
+public class IndexedItemModel
+{
+    public string? LanguageCode { get; set; }
+    public string? ClassName { get; set; }
```

Comment:
Why is this nullable? Every content item has a content type name.

Reply:
Changed